packages/e2e-tests/test-recipe-schema.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Sentry JavaScript E2E Test Recipe",
+  "type": "object",
+  "properties": {
+    "testApplicationName": {
+      "type": "string",
+      "description": "Name displayed in test output"
+    },
+    "buildCommand": {
```

Comment:
we should also add a timeout for build commands

Reply:
Added in 114ae0da64cbd4dc4159fe809a1ac23d646d98d7